content/tutorials/manage-the-rippled-server/troubleshooting/server-wont-start.md

```diff
@@ -0,0 +1,169 @@
+
+```text
+Loading: "/etc/opt/ripple/rippled.cfg"
+Failed to open '"/etc/opt/ripple/rippled.cfg"'.
+Terminating thread rippled: main: unhandled St13runtime_error 'Can not create "/var/opt/ripple"'
+Aborted (core dumped)
+```
+
+Possible solutions:
+
+- Check that `/etc/opt/ripple/rippled.cfg` exists and the `rippled` user has read permissions to the file. (Assuming you use the `rippled` user to run the `rippled` process, and you want to use the default location for the config file.)
+
+- Create a config file that can be read by the `rippled` user to `$HOME/.config/ripple/rippled.cfg` (where `$HOME` points to the `rippled` user's home directory).
+
+    **Tip:** The `rippled` repository contains [an example `rippled.cfg` file](https://github.com/ripple/rippled/blob/master/cfg/rippled-example.cfg) which is provided as the default config when you do an RPM installation. If you do not have the file, you can copy it from there.
+
+- Specify the path to your preferred config file using the `--conf` [commandline option](commandline-usage.html).
+
+## Failed to open validators file
+
+If `rippled` crashes on startup with an error such as the following, it means it can read its primary config file, but that config file specifies a separate validators config file (typically named `validators.txt`), which `rippled` cannot read.
+
+```text
+Loading: "/home/rippled/.config/ripple/rippled.cfg"
+Terminating thread rippled: main: unhandled St13runtime_error 'The file specified in [validators_file] does not exist: /home/rippled/.config/ripple/validators.txt'
+Aborted (core dumped)
+```
+
+Possible solutions:
+
+- Check that the `[validators.txt]` file exists and the `rippled` user has permissions to read it.
+
+    **Tip:** The `rippled` repository contains [an example `validators.txt` file](https://github.com/ripple/rippled/blob/master/cfg/validators-example.txt) which is provided as the default config when you do an RPM installation. If you do not have the file, you can copy it from there.
+
+- Edit your `rippled.cfg` file and modify the `[validators_file]` setting to have the correct path to your `validators.txt` (or equivalent) file. Check for extra whitespace before or after the filename.
+
+- Edit your `rippled.cfg` file and remove the `[validators_file]` setting. Add validator settings directly to your `rippled.cfg` file. For example:
+
+        [validator_list_sites]
+        https://vl.ripple.com
+
+        [validator_list_keys]
+        ED2677ABFFD1B33AC6FBC3062B71F1E8397C1505E1C42C64D11AD1B28FF73F4734
+
+
+## Cannot create database path
+
+If `rippled` crashes on startup with an error such as the following, it means the server does not have write permissions to the `[database_path]` from its config file.
+
+```text
+Loading: "/home/rippled/.config/ripple/rippled.cfg"
+Terminating thread rippled: main: unhandled St13runtime_error 'Can not create "/var/lib/rippled/db"'
```

Comment:
cannot

Reply:
Leaving as-is because the message originates in the code